cpp/src/groupby/sort/sort_helper.cu

```diff
@@ -106,7 +106,10 @@ size_type sort_groupby_helper::num_keys(cudaStream_t stream) {
 
 column_view sort_groupby_helper::key_sort_order(cudaStream_t stream) {
   if (_key_sorted_order)
-    return _key_sorted_order->view();
+    return cudf::experimental::detail::slice(
```

Comment:
If the behavior changed, I'm guessing some docs need to be updated?

Reply:
Updated in `sort_helper.hpp`